### PR TITLE
check dwProtocol for usb-ccid in linux guest

### PR DESCRIPTION
**Pull request by XueqiangWei**
id: 1775592
Signed-off-by: Xueqiang Wei <xuwei@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
(01/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_ccid.without_usb_hub.qemu-xhci: PASS (79.84 s)
 (02/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_ccid.without_usb_hub.qemu-xhci.q35: PASS (64.96 s)
 (03/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_hub.without_usb_hub.piix3-uhci: PASS (64.99 s)
 (04/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_hub.without_usb_hub.piix3-uhci.q35: PASS (63.58 s)
 (05/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_reboot.usb_tablet.without_usb_hub.usb-ehci: PASS (112.45 s)
 (06/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_reboot.usb_tablet.without_usb_hub.usb-ehci.q35: PASS (119.45 s)
 (07/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_reboot.usb_ccid.with_usb_hub.one_usb_dev.piix3-uhci: PASS (117.31 s)
 (08/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_reboot.usb_ccid.with_usb_hub.one_usb_dev.piix3-uhci.q35: PASS (117.25 s)
 (09/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_ccid.without_usb_hub.qemu-xhci: PASS (93.37 s)
 (10/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_ccid.without_usb_hub.qemu-xhci.q35: PASS (98.76 s)
 (11/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_hub.without_usb_hub.piix3-uhci: PASS (93.65 s)
 (12/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_hub.without_usb_hub.piix3-uhci.q35: PASS (97.44 s)
 (13/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.usb.usb_reboot.usb_tablet.without_usb_hub.usb-ehci: PASS (176.68 s)
 (14/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.usb.usb_reboot.usb_tablet.without_usb_hub.usb-ehci.q35: PASS (183.68 s)
 (15/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.usb.usb_reboot.usb_ccid.with_usb_hub.one_usb_dev.piix3-uhci: PASS (184.61 s)
 (16/16) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.usb.usb_reboot.usb_ccid.with_usb_hub.one_usb_dev.piix3-uhci.q35: PASS (187.16 s)

**Comment by XueqiangWei** [comment]
@vivianQizhu @ybduan please help review it. Many thanks.

File: qemu/tests/usb_basic_check.py [hunk 1]
```diff
@@ -72,6 +72,13 @@ def check_usb_device(test, params, env):
         exists, msgbox = check_usb_device_guest(*kargs)
         if not exists:
             test.fail(msgbox[0])
+    dwprotocols = params.get("dwprotocols", "")
```
Comment (ybduan) [inline qemu/tests/usb_basic_check.py]: params.get("dwprotocols")

**Comment by vivianQizhu** [comment]
Ack.

File: qemu/tests/usb_basic_check.py (hunk 1 above)
Comment (XueqiangWei) [reply qemu/tests/usb_basic_check.py]: agree, updated. thx

**Pull request merged**